downstream/modules/platform/ini/clustered-nodes.ini

```diff
@@ -0,0 +1,14 @@
+[controller]
+clusternode1.example.com
+clusternode2.example.com
+clusternode3.example.com
+
+[all:vars]
+admin_password='password'
+
+pg_host=''
+pg_port=''
+
+pg_database='awx'
+pg_username='awx'
+pg_password='password'
```

Comment:
Maybe this should be `<password>` to indicate a replaceable value. You could also go with `<changeme>` to make it more obvious that it's a value that the user needs to input and not a literal value.

Reply:
Noted about the replaceable value. Is there a reason why we'd be calling a seperate .ini file for the code block rather than including the code block in the main doc itself?

Reply:
There are a few reasons to externalize content to native file types. Doing this makes it easy to lint and validate automatically. For instance if you externalize YAML content to `yaml/myfile.yaml` you can lint it in your IDE to make sure the syntax is valid. You can also lint as part of the PR process. We could set things up to validate YAML, JSON, et al whenever someone pushes a commit.

Another good reason is reuse. You can include the same file in multiple modules. I'm hoping we'll get to the point where a lot of the upstream docs are externalizing files too and then we'll be able to include them in our asciidoc, if writers get into the habit of externalizing.

Another really compelling reason for externalizing files, is that engineers will be more likely to spot issues a piece of code in the correct format rather than embedded in asciidoc. Engineers, QE, and support teams are also more likely to contribute and help maintain externalized content and take responsibility for it. Writers are the SMEs for asciidoc but not necessarily for things like YAML or whatever else.